src/frontends/pytorch/src/op/avg_poolnd.cpp

```diff
@@ -68,7 +68,7 @@ OutputVector translate_avg_poolnd(const NodeContext& context) {
     }
 
     return {context.mark_node(
-        std::make_shared<v1::AvgPool>(input, strides, pads, pads, kernel, !count_include_pad, rounding_type))};
+        std::make_shared<v14::AvgPool>(input, strides, pads, pads, kernel, !count_include_pad, rounding_type))};
```

Comment:
Please remove AvgPool changes from this PR

Reply:
Done.